static/src/stylesheets/module/content/_content.global.scss

```diff
@@ -120,11 +120,9 @@
     }
 
     .meta__extras--crossword & {
```

Comment:
Does it seem odd that the crossword specific styles are not in the crossword stylesheets? I am not sure what the consistent approach is, modifications within the component or modifications in the feature stylesheets?